requirements.txt

```diff
@@ -19,3 +19,4 @@ configparser>=3.5.0
 openshift>=0.6
 pyasn1>=0.1.9
 pycrypto>=2.6.1
+pygithub>=1.43.5
```

Comment:
@14rcole: Instead of having pygithub a hard requirement for linchpin. Shouldn't we include it in Dockerfiles? Since containers are responsible for the release. Though, I am okay with adding pygithub as a requirement of linchpin. It bothers me a little because it is unrelated to the functionality of linchpin.

Reply:
That's a good point, I'll change that